src/pages/Transfer/CrossChainTransferForm/components/SelectChainInput/ChainInput.tsx

```diff
@@ -0,0 +1,41 @@
+import { forwardRef, useState } from 'react';
+
+import { ChainInputLabel } from './ChainInputLabel';
+import { ChainSelect } from './ChainSelect';
+
+type Chain = {
+  display: string;
+  id: string;
+};
+
+type Chains = Chain[];
+
+type Props = {
+  chains: Chains;
+};
+
+type ChainInputProps = Props;
+
+const ChainInput = forwardRef<HTMLInputElement, ChainInputProps>(
```

Comment:
This code could be on the `ChainSelect`

Reply:
Not sure I follow—do you mean removed this file altogether, and render the ChainSelect directly?

Reply:
I think overall the folder in not well structured.

I would not no this folder `SelectChainInput`, but instead `ChainSelect`, since we have a `TokenSelect` already.

I would not have `ChainInput`, because this is a `Select` component.

So overall the changes I would make would be merging `ChainInput` with `ChainSelect` and I would call it `ChainSelect`. I would than change the folder name to `ChainSelect`.

This component might have similarities with our `TokenInput` but they both end up being very different. `TokenInput` has an input and a select, meanwhile your component is simply a select.

Reply:
Yes, agree completely—dangers of copying and pasting. I'll check the account selector too, as that should follow the same input as this component, not `TokenSelect`